### ISSUE TITLE
bold or darken titles?

### ISSUE THREAD

**Issue by joshuago78** [gri-is/portal#534]
Joe suggests making the titles in the search results a little darker or bolder.

They are a fairly light color of blue and surrounded by a lot of white.

@kristenGRI take a second look and let me know what you think.

**Comment by kristenGRI** [comment]
@joshuago78 yeah I can see that. I think making the blue darker would be weird since we have an established color in place. However, I quickly mocked up two versions to see how you felt. 

Option A) I kept the link blue but I made the font a little bigger (16px instead of 14px) and gave it a font weight of 50  

Option B) I made the font color almost black (and gave it an underline so it was clear it was obvious that it was a link) and then made the View Digital Item button blue (since with the black text it looked too halloween-y with so much black and orange) I also kept the same font weight and size increase that appears in option A.

let me know your thoughts.

![option-a](https://cloud.githubusercontent.com/assets/15019503/15978665/83b77a10-2f14-11e6-93a3-0e68ca412468.png)

![option-b](https://cloud.githubusercontent.com/assets/15019503/15978672/90de1082-2f14-11e6-8740-07f2d1920eaa.png)
